.gitignore

```diff
@@ -2,4 +2,4 @@
 */target
 **/*.rs.bk
 Cargo.lock
-
+/.vscode
```

Comment:
In general, I'm not for including environment specific rules in the published `.gitignore`. Otherwise, it would rapidly be polluted with all the config directories of all possible editors used by different contributors. I advice that you put instead a `.gitignore` file *inside* your `.vscode` directory, containing `*`.

Reply:
Sry this was more an accident as on purpose I'll remove it with the next commit.